HISTORY.md

```diff
@@ -22,6 +22,7 @@
 
 ### Public API Changes
 * Substantial changes have been made to the Cache class to support internal development goals. Direct use of Cache class members is discouraged and further breaking modifications are expected in the future. SecondaryCache has some related changes and implementations will need to be updated. (Unlike Cache, SecondaryCache is still intended to support user implementations, and disruptive changes will be avoided.) (#10975)
+* Add `MergeOperationOutput::status` for merge operator users to report a specific failure type
```

Comment:
Can we explain what the recommendation we have to existing merge operator users?

Reply:
Sure, existing use cases are recommended to make no change so it will behave the same as before. I will reassure users they don't need to change any API usage assuming they don't want to change how failures are handled

Reply:
Done